react/ProductSummaryList.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { useQuery } from 'react-apollo'
 import productsQuery from 'vtex.store-resources/QueryProducts'
+import { ProductList } from 'vtex.structured-data'
```

Comment:
Minor nitpick, but could you rename this `ProductList` import to something a bit more explicit, like `ProductListStructuredData`? I know it becomes very verbose, but it's easier not to mistake this for a component that renders something to the screen on line 94, where this is used as a component.